loafwallet/src/WalletManager.swift

```diff
@@ -64,10 +64,11 @@ class WalletManager: BRWalletListener, BRPeerManagerListener {
 		return lazyPeerManager
 	}
 
+	// TODO: Pass the fpRate from User Preferences
 	internal lazy var lazyPeerManager: BRPeerManager? = {
 		guard let wallet = self.wallet else { return nil }
 		return BRPeerManager(wallet: wallet, earliestKeyTime: self.earliestKeyTime,
-		                     blocks: self.loadBlocks(), peers: self.loadPeers(), listener: self)
+		                     blocks: self.loadBlocks(), peers: self.loadPeers(), listener: self, falsePositivesRate: 0.00001)
```

Comment:
@Iferencak sorry for the terrible response time. fpRate tells the node to send a percentage of erroneous addresses to Litewallet during sync so that the sent `cloud` is not related to closely to the users actual address. 

The upside is more anonymity (higher rate) the downside is longer time to sync.